include/seqan/sequence/string_set_base.h

```diff
@@ -1802,6 +1804,25 @@ write(TTarget & target, TSequences const & seqs)
     write(target, seqs, '\n');
 }
 
+// ----------------------------------------------------------------------------
+// Function append() (works on any container of container)
+// ----------------------------------------------------------------------------
+
+template <typename TSequences1, typename TSequences2, typename TExpand >
+inline SEQAN_FUNC_ENABLE_IF(And<And<Is<ContainerConcept<TSequences1> >,
+                                    Is<ContainerConcept<typename Value<TSequences1>::Type > > >,
+                                And<Is<ContainerConcept<TSequences2> >,
+                                    Is<ContainerConcept<typename Value<TSequences2>::Type > > > >, void)
+append(TSequences1 & me, TSequences2 const & obj, Tag<TExpand>)
+{
+    typedef typename Iterator<TSequences2 const>::Type TSourceIt;
+
+    unsigned oldLength = length(me);
+    resize(me, oldLength + length(obj), Tag<TExpand>());
+    for (TSourceIt it = begin(obj), itEnd = end(obj); it != itEnd; ++it)
```

Comment:
Because it might not be a sequential container at all?
[this is the most generic case, owner stringsets have specializations]